cpp/src/community/nvgraph_gdf.cu

```diff
@@ -372,7 +365,7 @@ void triangle_count_nvgraph(Graph* G, uint64_t* result) {
 
 void louvain(Graph *graph, void *final_modularity, void *num_level, void *louvain_parts_ptr, int max_iter) {
 
-  CUGRAPH_EXPECTS(graph->adjList != nullptr, "Invalid API parameter");
+  CUGRAPH_EXPECTS(graph->adjList != nullptr, "Invalid API parameter: graph adjList is NULL");
```

Comment:
Why aren't we using CHECK_GRAPH(graph) here?

Reply:
fixed.